CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -123,95 +209,170 @@ private function createDirectDebitPaymentProcessorType() {
    */
   private function createDirectDebitPaymentProcessor() {
     $paymentProcessor = [
-      'name' => ts('Direct Debit'),
+      'name' => 'Direct Debit',
       'description' => '',
       'payment_processor_type_id' => 'OfflineDirectDebit',
       'domain_id' => CRM_Core_Config::domainID(),
-      'is_active' => 1,
       'payment_instrument_id' => 'direct_debit',
+      'is_active' => 1,
     ];
     civicrm_api3('PaymentProcessor', 'create', $paymentProcessor);
   }
 
+  public function onEnable() {
+    $this->alterNavigationMenu("direct_debit", "enable");
```

Comment:
Since you're altering navigation menu, you should call `CRM_Core_BAO_Navigation::resetNavigation();`.